.github/ISSUE_TEMPLATE/new-sample-setup.md

```diff
@@ -9,15 +9,14 @@ assignees: ''
 
 ## New Sample Checklist
 
-<!-- markdown-link-check-disable-next-line -->
-Ensure you've read the [Contributing Guide (Microsoft Internal only.)](https://dev.azure.com/csedtd/_git/DataOps?path=%2FCONTRIBUTING.md&_a=preview)
+Ensure you've read the [Contributing Guide](/CONTRIBUTING.md).
 
 ## Tooling
 
 - [ ] Setup/Configure Azure Subscriptions (if applicable)
 - [ ] CI/CD Tooling (Azure DevOps)
-- [ ] Setup Work Item tracking Tooling (GitHub Project)
-- [ ] Collaboration and Communication Tooling  (Teams)
+- [ ] Setup Work Item tracking Tooling (ie. GitHub Projects - Microsoft Employees only)
+- [ ] Collaboration and Communication Tooling  (ie. Teams)
 - [ ] Plan to create Devcontainer with appropriate linters and markdown link checkers.
 
 ## Planning
```

Comment:
Please fix all of these:

- [ ] Decide whether you are creating an end-to-end sample or single-technology sample.
- [ ] Decide which DataOps key concepts (e.g., IaC, CI/CD, Testing, Observability) will you be demonstrating in the sample.
- [ ] Decide which Well-Architected Framework concepts your sample will be demonstrating.
- [ ] Decide on the name of the sample and located where in the repository structure your sample will sit.
- [ ] Overall business / industry scenario of the sample identified (e.g., City Parking Sensor tracking solution) if applicable.

Reply:
Done